### PR TITLE
Update vrc files

### PR DESCRIPTION
**Pull request by haneslinger**
#### Any background context you want to provide?

I only manually changed `seed/tests/test_utils_geocode.py`. The rest auto generated when I deleted the old vcr files and ran the tests.

#### What's this PR do?

The old vcr files didn't reflect the current state of the api. I guess a building moved a little 🤷‍♀️ 
Here, I update the files and excepted results. 

#### How should this be manually tested?

On develop, if you remove the vcr files and run `python manage.py test seed/tests/test_utils_geocode.py` the tests will fail, here they won't. 

#### What are the relevant tickets?

https://github.com/SEED-platform/seed/issues/3095

### CODE REVIEW

File: seed/tests/data/vcr_cassettes/geocode_base_case.yaml [hunk 1]
```diff
@@ -9,52 +9,50 @@ interactions:
...
       - keep-alive
       User-Agent:
-      - python-requests/2.20.0
+      - python-requests/2.22.0
     method: GET
     uri: https://www.mapquestapi.com/geocoding/v1/batch?inFormat=json&json=%7B%22locations%22%3A+%5B%7B%22street%22%3A+%223001+Brighton+Blvd%2C+suite+2693%2C+Denver%2C+Colorado%2C+80216%22%7D%5D%7D&maxResults=2&outFormat=json&thumbMaps=false
   response:
     body:
-      string: '{"info":{"statuscode":0,"copyright":{"text":"\u00A9 2019 MapQuest,
+      string: '{"info":{"statuscode":0,"copyright":{"text":"\u00A9 2022 MapQuest,
```
Comment (nllong) [inline seed/tests/data/vcr_cassettes/geocode_base_case.yaml:17]: nice, this may explain the subtle differences/resolutions.

**Review by nllong — APPROVED** [review]
looks great!

**Comment by nllong** [comment]
@macintoshpie  -- want to review real quick, then merge if all good?

**Review by macintoshpie — APPROVED** [review]
👍

**Comment by axelstudios** [comment]
@haneslinger Test

**Pull request merged**